resip/recon/ConversationManager.hxx

```diff
@@ -483,6 +486,18 @@ public:
    virtual void onDtmfEvent(ParticipantHandle partHandle, int dtmf, int duration, bool up) = 0;
 
    ///////////////////////////////////////////////////////////////////////
+   // Pager Message Handlers /////////////////////////////////////////////
+   ///////////////////////////////////////////////////////////////////////
+
+   virtual void onMessageArrived(resip::ServerPagerMessageHandle handle, const resip::SipMessage& message) = 0;
```

Comment:
Yes that's correct.  Handling in-dialog MESSAGE requests is simply handling
the onMessage API from InviteSessionHandler.  You don't create a dialog
specifically for MESSAGE handling - but you could send an in-dialog MESSAGE
on and existing call using the InviteSession::message API.  I'm not sure
you should worry about in-dialog MESSAGEs at this point, if you don't have
a direct need for them.